packages/contract-tests/test/methods.js

```diff
@@ -589,40 +589,61 @@ describe("Methods", function () {
     });
   });
 
-  describe("sendTransaction() / send() [ @geth ]", function () {
+  describe("sendTransaction() / call() / estimateGas() / send() [ @geth ]", function () {
     it("should trigger the fallback function when calling sendTransaction()", async function () {
       const example = await Example.new(1);
-      const triggered = await example.fallbackTriggered();
+      let triggered = await example.fallbackTriggered();
 
-      assert(
-        triggered === false,
-        "Fallback should not have been triggered yet"
-      );
+      assert.isFalse(triggered, "Fallback should not have been triggered yet");
 
       await example.sendTransaction({
         value: web3.utils.toWei("1", "ether")
       });
 
       const balance = await web3.eth.getBalance(example.address);
-      assert(
-        balance === web3.utils.toWei("1", "ether"),
+      assert.strictEqual(
+        balance,
+        web3.utils.toWei("1", "ether"),
         "Balance should be 1 ether"
       );
+
+      triggered = await example.fallbackTriggered();
+      assert.isTrue(triggered, "Fallback should now have been triggered");
     });
 
     it("should trigger the fallback function when calling send() (shorthand notation)", async function () {
       const example = await Example.new(1);
-      const triggered = await example.fallbackTriggered();
+      let triggered = await example.fallbackTriggered();
 
-      assert(
-        triggered === false,
-        "Fallback should not have been triggered yet"
-      );
+      assert.isFalse(triggered, "Fallback should not have been triggered yet");
 
       await example.send(web3.utils.toWei("1", "ether"));
 
       const balance = await web3.eth.getBalance(example.address);
-      assert(balance === web3.utils.toWei("1", "ether"));
+      assert.strictEqual(balance, web3.utils.toWei("1", "ether"));
+
+      triggered = await example.fallbackTriggered();
+      assert.isTrue(triggered, "Fallback should now have been triggered");
+    });
+
+    it("should trigger the fallback function when calling call()", async function () {
+      const example = await Example.new(1);
+
+      const result = await example.call({ data: "0x01234567" });
```

Comment:
Can you explain the meaning of the value of `data` here?

Reply:
It's the binary data passed to the call, just like in a `instance.sendTransaction`.  This one is obviously just an example.  I made it so that the fallback function takes the passed in data and cuts off the second half, so I passed in `0x01234567` and checked that I got `0x0123` back.  (Since the 4 bytes `0x01234567` doesn't correspond to any function selector, it'll hit the fallback function rather than any of the contract's defined functions.)

Does that answer your question?